docs/dev/plugins.rst

```diff
@@ -238,6 +238,11 @@ The events currently available are:
   during a ``beet import`` interactive session. Plugins can use this event for
   :ref:`appending choices to the prompt <append_prompt_choices>` by returning a
   list of ``PromptChoices``. Parameters: ``task`` and ``session``.
+  
+* `extracting_trackdata` and `extracting_albumdata`: called after the metadata
+  is obtained from MusicBrainz. The parameter is a ``dict`` containing the data
+  retrieved from MusicBrainz for a track or an album.
+  Allows to add additional tags.
```

Comment:
The prevailing style on this page seems to be to give every event its own bullet, even if they're similar.

Reply:
We should also give the name of the parameter.